HSLdevcom/helmet-ui
# Add scenario duplication functionality, add copy icon

How to test:
- Pull branch , npm install, npm start
- Click the duplication button next to scenario on the scenario list
- That scenario should be duplicated, settings transferred and name changed to original name + shortened unique UUID of the new scenario (in this case the duplicate)
